### PR TITLE
SEAR-129 Spike: Show/Hide bottom pagination based on number of search results 

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-129

#### Brief feature description
- Added conditional pagination dropdown based on the scrollbar

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
-n/a

### CODE REVIEW

File: app/javascript/search/common/commonUtils.js [hunk 1]
```diff
@@ -86,6 +86,10 @@ export const checkNullOrEmptyValue = (value) => {
   return (value === null || value === '') ? 'N/A' : value
 }
 
+export const checkIfScrollBarVisible = (object) => {
```
Comment (ratneshraval) [inline app/javascript/search/common/commonUtils.js]: This isn't search specific. Can we move this function to more general file?

Comment (Nasir2712) [reply app/javascript/search/common/commonUtils.js]: Haven't used it anywhere else but except here. So, that's why had it here in search specific common utils.

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by ratneshraval** [comment]
I like the functionality and it works pretty well. Although I suggest we get this reviewed by PO.

My question is:
1. Should it decide on pagination when component updates
OR
2. Should it show/hide based on window resize, without component update.

**Comment by Nasir2712** [comment]
The pagination is decided when the component updates. The pagination is dependent on Search Results. It cannot show/hide on window resize without a component update because whenever search results update it should check the presence of the scrollbar.

**Pull request merged**